packages/trip-details/src/index.tsx

```diff
@@ -325,58 +309,20 @@ export function TripDetails({
                   }
                   description="Text stating that portions of the trip include a flex (on-demand) transit service."
                   id="otpUi.TripDetails.tripIncludesFlex"
+                  values={{
+                    extraMessage: [
```

Comment:
Please render `extraMessage` as its own JSX after `<FormattedMessage>`. The reason is that it is content that is completely independent (although related) of the sentence "This trip includes flex routes."

Reply:
See previous comment about RTL langauges